scripts/_utils.py

```
@@ -198,62 +196,10 @@ def count_repo_lines(repo_path: str | None = None) -> int:
 
 def remove_path(path: str) -> None:
-    """
-    Remove a file or directory using OS-native fast deletion.
-
-    Uses ``cmd /c rd /s /q`` on Windows and ``rm -rf`` on Unix,
-    falling back to ``shutil.rmtree`` on failure.
-
-    :param path: Path to the file or directory to remove.
-    """
-    if not os.path.exists(path):
+    p = Path(path)
```

Ruff (0.15.17)

[error] Undefined name `Path`

(F821)